packages/components/Molecules/Card/CardPromo.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <rpl-link class="rpl-card-promo" :class="modifiers" :href="link.url">
```

Comment:
We should keep `rpl-card-promo` class with modifiers if we follow `BEM`. See examples: http://getbem.com/naming/

Reply:
I've moved this to the modifiers() as it confuses me having two `class=""`. I'll put it back here so it's more clear that it's only for modifiers 👍